Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -53,6 +53,10 @@ theorem logb_zero : logb b 0 = 0 := by simp [logb]
 theorem logb_one : logb b 1 = 0 := by simp [logb]
 #align real.logb_one Real.logb_one
 
+@[simp]
+lemma logb_self_eq_one (hb : 1 < b) : logb b b = 1 :=
```

Comment:
I think this can be strengthened to `0 < b` and `b != 1`?

Reply:
Indeed; I added this stronger version as a separate lemma, since proving `1 < b` is probably what we want to do most of the time.